docs/source/references/config.md

```diff
@@ -8,11 +8,8 @@ The following is the default configuration used by Iroha.
 
 ```json
 {
-  "PUBLIC_KEY": "ed01201c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b",
-  "PRIVATE_KEY": {
-    "digest_function": "ed25519",
-    "payload": "282ed9f3cf92811c3818dbc4ae594ed59dc1a2f78e4241e31924e101d6b1fb831c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b"
-  },
+  "PUBLIC_KEY": null,
+  "PRIVATE_KEY": null,
```

Comment:
In the beginning of the section we say that this is default iroha configuration, but this config is non working, because it has missing fields. Is this ok?

Reply:
There's still quite some work that needs to be done on kagami docs, and there are quite a few places I don't like about the new `config.md`, such as mentioning the types of `ConfigurationProxy`. At the very least, it needs a small explanation at the start that any fields shown as `null` should be user provided. At best, come up with a new workaround for `kagami` instead of deriving `Documented` on `Proxy`. But as for latter, I'm a bit at my wit's end for now.

Reply:
Just adding a note at the top of kagami docgen in the last push. Surely a makeshift measure.